testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/forms/CustomFlowTest.java

```diff
@@ -270,37 +266,33 @@ public void validateX509FlowUpdate() throws Exception {
         flow.setTopLevel(true);
         flow.setBuiltIn(false);
 
-        try {
-            String executionId;
-            try (Response response = authMgmtResource.createFlow(flow)) {
-                Assert.assertThat("Create flow", response, statusCodeIs(Response.Status.CREATED));
-                AuthenticationFlowRepresentation newFlow = findFlowByAlias(flowAlias);
-                flowId = newFlow.getId();
-            }
-
-            //add execution - username-password form
-            Map<String, String> data = new HashMap<>();
-            data.put("provider", ValidateX509CertificateUsernameFactory.PROVIDER_ID);
-            authMgmtResource.addExecution(flowAlias, data);
+        try (Creator.Flow amr = Creator.create(testRealm(), flow)) {
+            AuthenticationManagementResource authMgmtResource = amr.resource();
 
-            List<AuthenticationExecutionInfoRepresentation> executions = authMgmtResource.getExecutions(flowAlias);
-            assertThat(executions, hasSize(1));
-            final AuthenticationExecutionInfoRepresentation execution = executions.get(0);
-            executionId = execution.getId();
+            //add execution - X509 username
+            final AuthenticationExecutionInfoRepresentation execution = amr.addExecution(ValidateX509CertificateUsernameFactory.PROVIDER_ID);
+            String executionId = execution.getId();
 
             Map<String, String> config = new HashMap<>();
             config.put(AbstractX509ClientCertificateAuthenticator.ENABLE_CRL, Boolean.TRUE.toString());
             AuthenticatorConfigRepresentation authConfig = new AuthenticatorConfigRepresentation();
             authConfig.setAlias("Config alias");
             authConfig.setConfig(config);
 
+            String acId;
             try (Response resp = authMgmtResource.newExecutionConfig(executionId, authConfig)) {
                 assertThat(resp, statusCodeIs(Status.CREATED));
+                acId = ApiUtil.getCreatedId(resp);
             }
-        } finally {
-            if (flowId != null) {
-                authMgmtResource.deleteFlow(flowId);
-            }
+
+            authConfig = authMgmtResource.getAuthenticatorConfig(acId);
+            authConfig.getConfig().put(AbstractX509ClientCertificateAuthenticator.ENABLE_CRL, Boolean.FALSE.toString());
+            authConfig.getConfig().put(AbstractX509ClientCertificateAuthenticator.CRL_RELATIVE_PATH, "");
+
+            authMgmtResource.updateAuthenticatorConfig(acId, authConfig);
+
+            // Saving the same options for the second time would fail for CRL_RELATIVE_PATH on Oracle due to "" == NULL weirdness
```

Comment:
Very minor nitpicking: This comment is now not true anymore as with the fix, it wouldn't fail anymore. Maybe slightly updating this comment (and possibly add reference to JIRA) could be fine?